### ISSUE TITLE
Unable to handle action installSPFXSolution

### ISSUE THREAD

**Issue by Chaitra1006** [SharePoint/sp-dev-site-scripts#8]
"Add-SPOSiteScript : Unable to handle action installSPFXSolution. No action handler was registered for this action." error is encountered on Add-SPOSiteScript execution.

Whereas Site Script with verb:applyTheme works perfectly fine.

![image](https://user-images.githubusercontent.com/38909384/39517323-2dbe7792-4e1d-11e8-9f20-b23fa3ccc61f.png)

Am I missing any step? What could be the issue?

Thank you for reporting an issue or suggesting an enhancement. We appreciate your feedback - to help the team to understand your needs, please complete the below template to ensure we have the necessary details to assist you. 

#### Category
- [ ] Question
- [ #] Bug
- [ ] Enhancement

#### Expected or Desired Behavior
_If you are reporting a bug, please describe the expected behavior. If you are suggesting an enhancement please describe thoroughly the enhancement, how it can be achieved, and expected benefit._

#### Observed Behavior
_If you are reporting a bug, please describe the behavior you expected to occur when performing the action. If you are making a suggestion, you can delete this section._

#### Steps to Reproduce
_If you are reporting a bug please describe the steps to reproduce the bug in sufficient detail to allow testing. Only way to fix things properly, is to have sufficient details to reproduce it. If you are making a suggestion, you can delete this section._

#### Submission Guidelines
_Delete this section after reading_
- All suggestions or bugs are welcome, please let us know what's on your mind.
- If you are reporting an issue around any of the samples, please ensure that you have clear reference on the sample and possibly code file, which should be fixed.
- Remember to include sufficient details and context.
- If you have multiple suggestions or bugs please submit them in separate bugs so we can track resolution.

Thanks for your contribution! Sharing is caring.

**Comment by dhavalshah01** [comment]
I am facing the same issue. I even installed new SP Online management shell. but looks like its not taking the verb installSPFXSolution. If I remove the "installSPFXSolution" node, it creates the site design. 

Is the support to installSPFXSolution is still not available ?

Was anyone able to resolve it ?

**Comment by morten-fresh** [comment]
Seeing the same issue. Any updates on this? At least an acknowledgement that this is an actual bug og just a rollout delay.  Other actions from the April Release is working just fine.

**Comment by christianteh** [comment]
use installSolution instead of installSPFXSolution

**Comment by dhavalshah01** [comment]
@christianteh  yes you are right, the syntax has changed to installSolution. That solved the issue.
